packages/steveo/src/producers/sqs.ts

```diff
@@ -101,6 +114,82 @@ class SqsProducer extends BaseProducer implements IProducer {
     return res.QueueUrl;
   }
 
+  async getDeadLetterQueuePolicy(
```

Comment:
Is this the same code as above? Should we have just one place where the queue creation code lives?

Reply:
@sugendran I'm not sure if I captured that correctly, do we remove the queue creation on the producer initialization and only have it on the consumer createQueue? or it's because of redundancy, I think I can move this on a util which we currently don't have yet so might be a specific one for SQS.

Reply:
@sugendran sorry I just get it now, so we remove the DLQ checking on the producer side and let it be created on the queue creation on the consumer side?

Reply:
But I feel like we needed to have on both for good measurements. Will just find a way to make the code not redundant

Reply:
If it's not possible to remove the redundancy that's fine, but the ideal case is that we have one set of logic for creating the DLQ so that there are no differences whether the producer or the consumer creates it.

Reply:
@sugendran I'm exploring a solution using a mixin function wherein I create a function that returns a class that extends to either BaseProducer or BaseConsumer (typing is preserved through generics)